simulator/util/dummy_perception_publisher/package.xml

```diff
@@ -14,6 +14,7 @@
   
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
+  <test_depend>ament_lint_auto</test_depend>
```

Comment:
`ament_lint_auto` is declared twice, please remove this one.

Reply:
This is resolved